### DISCUSSION TITLE
Support mapping to relationships from tree tables in WB

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5582] [Feature Requests]
modifiedBy is not appearing under the Taxon table, even with hidden fields displaying, in the WB mapper.

**EDIT:** No relationships are appearing, so no Agent or Taxon Attributes

Here is the entire HTML section:

<section class="bg-[color:var(--custom-select-b1)] flex flex-col
        custom-select-group-hidden-fields" role="group" aria-label="Hidden Fields"><header aria-hidden="true" class="bg-[color:var(--custom-select-b2)] px-1 cursor-auto">Hidden Fields</header><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="ID  " aria-label="ID  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">ID</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="CITES Status  " aria-label="CITES Status  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">CITES Status</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Col Status  " aria-label="Col Status  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Col Status</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Cultivar Name  " aria-label="Cultivar Name  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Cultivar Name</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Environmental Protection Status  " aria-label="Environmental Protection Status  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Environmental Protection Status</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="ESA Status  " aria-label="ESA Status  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">ESA Status</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Group Number  " aria-label="Group Number  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Group Number</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Highest Child Node Number  " aria-label="Highest Child Node Number  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Highest Child Node Number</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Integer2  " aria-label="Integer2  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Integer2</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Integer3  " aria-label="Integer3  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Integer3</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Integer4  " aria-label="Integer4  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Integer4</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Integer5  " aria-label="Integer5  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Integer5</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Is Hybrid  " aria-label="Is Hybrid  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Is Hybrid</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="ISIS Number  " aria-label="ISIS Number  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">ISIS Number</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Label Format  " aria-label="Label Format  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Label Format</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="LSID  " aria-label="LSID  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">LSID</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="NCBI Taxon Number  " aria-label="NCBI Taxon Number  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">NCBI Taxon Number</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Node Number  " aria-label="Node Number  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Node Number</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Number 1  " aria-label="Number 1  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Number 1</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Number 2  " aria-label="Number 2  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Number 2</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Number3  " aria-label="Number3  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Number3</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Number4  " aria-label="Number4  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Number4</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Number5  " aria-label="Number5  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Number5</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Rank ID  " aria-label="Rank ID  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Rank ID</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Taxonomic Serial Number  " aria-label="Taxonomic Serial Number  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Taxonomic Serial Number</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="text10  " aria-label="text10  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">text10</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="S  " aria-label="S  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">S</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Text12  " aria-label="Text12  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Text12</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Text13  " aria-label="Text13  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Text13</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Text16  " aria-label="Text16  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Text16</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="text6  " aria-label="text6  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">text6</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Text7  " aria-label="Text7  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Text7</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Text8  " aria-label="Text8  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Text8</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Text9  " aria-label="Text9  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Text9</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 !cursor-not-allowed text-gray-500 bg-[color:var(--custom-select-b1)] custom-select-option-selected cursor-auto bg-[color:var(--custom-select-accent)]" title="Timestamp Modified  (Selected)" aria-label="Timestamp Modified  (Selected)" tabindex="-1" aria-selected="true" role="option" aria-disabled="true" aria-current="true" aria-atomic="true"><span class="w-table-icon h-table-icon flex items-center justify-center font-bold text-green-500" aria-label="Mapped" role="img"><svg class="w-6 h-6" aria-hidden="true" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" d="M16.707 5.293a1 1 0 010 1.414l-8 8a1 1 0 01-1.414 0l-4-4a1 1 0 011.414-1.414L8 12.586l7.293-7.293a1 1 0 011.414 0z" clip-rule="evenodd"></path></svg></span><span class="flex-1">Timestamp Modified</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Unit Ind1  " aria-label="Unit Ind1  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Unit Ind1</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Unit Ind2  " aria-label="Unit Ind2  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Unit Ind2</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Unit Ind3  " aria-label="Unit Ind3  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Unit Ind3</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Unit Ind4  " aria-label="Unit Ind4  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Unit Ind4</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Unit Name1  " aria-label="Unit Name1  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Unit Name1</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Unit Name2  " aria-label="Unit Name2  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Unit Name2</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Unit Name3  " aria-label="Unit Name3  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Unit Name3</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Unit Name4  " aria-label="Unit Name4  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Unit Name4</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="USFWS Code  " aria-label="USFWS Code  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">USFWS Code</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Version  " aria-label="Version  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Version</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Visibility  " aria-label="Visibility  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Visibility</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="yesNo12  " aria-label="yesNo12  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">yesNo12</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Yes No13  " aria-label="Yes No13  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Yes No13</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Yes No14  " aria-label="Yes No14  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Yes No14</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Yes No15  " aria-label="Yes No15  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Yes No15</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Yes No16  " aria-label="Yes No16  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Yes No16</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Yes No17  " aria-label="Yes No17  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Yes No17</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Yes No18  " aria-label="Yes No18  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Yes No18</span><span class="print:hidden w-6 h-6"></span></span><span class="p-1 flex items-center gap-x-1 hover:bg-[color:var(--custom-select-b2)] focus:bg-[color:var(--custom-select-b2)]" title="Yes No19  " aria-label="Yes No19  " tabindex="-1" aria-selected="false" role="option" aria-disabled="false" aria-current="false" aria-atomic="true"><span class="w-table-icon h-table-icon" aria-hidden="true"></span><span class="flex-1">Yes No19</span><span class="print:hidden w-6 h-6"></span></span></section>

**Comment by grantfitzsimmons** [comment]
![image](https://user-images.githubusercontent.com/37256050/167048662-eaa77347-322a-4113-af4c-584fd735f5de.png)

**Comment by maxpatiiuk** [comment]
All relationships are removed from tree tables as WorkBench does not currently support uploading to relationships from tree tables

**Comment by maxpatiiuk** [comment]
For reference, you also can not map to -to-many relationships inside of -to-many relationships

**Comment by grantfitzsimmons** [comment]
@maxpatiiuk Gotcha. What is the technical issue with WB uploading to tree tables?

**Comment by grantfitzsimmons** [comment]
Andy had another scenario where this ability would be very helpful for KU Mammals.

The WB should support a -to-many mapping to a tree table!

**Comment by benanhalt** [comment]
The prep storage thing? This wouldn't help with that.

**Comment by grantfitzsimmons** [comment]
This would enable users to map a species hybrid in the WorkBench if it was resolved!

**Comment by grantfitzsimmons** [comment]
https://discourse.specifysoftware.org/t/workbench-map-hybrid-species-how/1261/1

This is a use case for why we should add support for relationships from tree tables!

**Comment by specifysoftware** [comment]
This issue has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/workbench-map-hybrid-species-how/1261/8
